R/render_site.R

```diff
@@ -204,9 +204,9 @@ render_site <- function(input = ".",
 
 #' @rdname render_site
 #' @param preview Whether to list the files to be removed rather than actually
-#'   removing them.
+#'   removing them. Defaulting to TRUE to prevent removing without notice.
 #' @export
-clean_site <- function(input = ".", preview = FALSE, quiet = FALSE,
+clean_site <- function(input = ".", preview = TRUE, quiet = FALSE,
```

Comment:
Would it be interesting to have an option for `preview` to change the default behavior of the IDE button to FALSE ? 

Not so sure, so I did not add it

Reply:
Yes, that could be useful (e.g., `preview = getOption('rmarkdown.clean_site.preview', TRUE)`), but let's wait until someone asks for it.

Reply:
Oh I remember this method of yours now - not used to that: get ideas but wait for people asking ! 😄

Reply:
You can definitely also proactively ask if anyone wants this feature in advance. I just don't have the energy to do this myself :)